.github/actions/spell-check/whitelist.txt

```diff
@@ -0,0 +1,3240 @@
+FFDE
+FFF
+fffd
+FFFDu
+ffff
+ffffff
+FFFFFFFF
+FFFFFFFFFFFF
+FFrom
+ffu
+FGCOLOR
+fgetc
+fgetwc
+fgidx
+FILEDESCRIPTION
+fileno
+FILEPATH
+FILESUBTYPE
+FILESYSPATH
+filesystem
+FILETIME
+FILETYPE
+FILEW
+fililng
+FILLATTR
+FILLCONSOLEOUTPUT
+Filledbox
+FILTERONPASTE
+finalizer
+FINDCASE
+FINDDLG
+FINDDOWN
+finden
+findlocalename
+FINDSTR
+FINDSTRINGEXACT
+FINDUP
+Firefox
+FIter
+FIXEDCONVERTED
+Flg
+flyout
+fmodern
+fmtarg
+fmtid
+FOLDERID
+FONTCHANGE
+fontdlg
+FONTENUMDATA
+FONTENUMPROC
+FONTFACE
+FONTFAMILY
+FONTHEIGHT
+fontlist
+FONTOK
+FONTSIZE
+FONTSTRING
+fonttbl
+FONTTYPE
+FONTWEIGHT
+FONTWIDTH
+FONTWINDOW
+forceinline
+FORCEOFFFEEDBACK
+FORCEONFEEDBACK
+FORCEV
+foreach
+fprintf
+framebuffer
+FRAMECHANGED
+fre
+freopen
+frontend
+Fscreen
+FSCTL
+FSINFOCLASS
+fsproj
+fstream
+fte
+Ftm
+fullcolor
+fullscreen
+fullwidth
+func
+FUNCTIONCALL
+fuzzer
+fuzzmap
+fuzzwrapper
+fwdecl
+fwe
+fwlink
+fyi
+Gabriola
+Gatta
+gb
+gci
+gcx
+gcy
+gdi
+gdip
+gdirenderer
+GENERATEPROJECTPRIFILE
+geopol
+GETALIAS
+GETALIASES
+GETALIASESLENGTH
+GETALIASEXES
+GETALIASEXESLENGTH
+getboundingrectangles
+GETCARETWIDTH
+getch
+getchar
+GETCOMMANDHISTORY
+GETCOMMANDHISTORYLENGTH
+GETCONSOLEINPUT
+GETCONSOLEPROCESSLIST
+GETCONSOLEWINDOW
+GETCOUNT
+GETCP
+GETCURSEL
+GETCURSORINFO
+GETDISPLAYMODE
+GETDISPLAYSIZE
+GETDLGCODE
+GETDPISCALEDSIZE
+GETFONTINFO
+GETFONTSIZE
+getglyphs
+GETHARDWARESTATE
+GETHUNGAPPTIMEOUT
+GETICON
+GETITEMDATA
+GETKEYBOARDLAYOUTNAME
+GETKEYSTATE
+GETLARGESTWINDOWSIZE
+GETLBTEXT
+getline
+getlocalename
+getlocalenamelength
+GETMINMAXINFO
+GETMOUSEINFO
+GETNUMBEROFFONTS
+GETNUMBEROFINPUTEVENTS
+GETOBJECT
+GETPOS
+GETSELECTIONINFO
+getset
+getstring
+getstringlength
+GETTEXT
+GETTEXTLEN
+GETTITLE
+GETWAITTOKILLSERVICETIMEOUT
+GETWAITTOKILLTIMEOUT
+GETWHEELSCROLLCHARACTERS
+GETWHEELSCROLLCHARS
+GETWHEELSCROLLLINES
+getwriter
+Gfun
+gfx
+ggggg
+gggggg
+ggggggg
+gh
+github
+githubusercontent
+gle
+globals
+gmail
+GMEM
+GNUC
+Goldmine
+gonce
+Google
+goutput
+Gravell's
+grayscale
+GREENSCROLL
+Grehan
+Greyscale
+gridline
+Grie
+Griese
+groupbox
+gsl
+GTP
+guc
+gui
+guidatom
+guidgenerator
+GValue
+gw
+GWL
+GWLP
+gwsz
+HABCDEF
+halfwidth
+HALTCOND
+HANGEUL
+hardcoded
+hardcodes
+hashalg
+HASSTRINGS
+hbitmap
+hbm
+HBMMENU
+hbmp
+hbr
+hbrush
+hc
+HCmd
+hdc
+hdr
+HDROP
+hdrstop
+HEIGHTSCROLL
+Hernan
+herunterzuladen
+hfile
+hfont
+hglobal
+hh
+hhc
+hhk
+hhook
+hhp
+hhx
+HIBYTE
+HICON
+HIDEWINDOW
+hidpi
+HIGHLIGHTTEXT
+hinst
+HINSTANCE
+Hirots
+HISTORYBUFS
+HISTORYNODUP
+HISTORYSIZE
+hittest
+HIWORD
+hk
+HKCU
+hkey
+hkl
+HKLM
+hlocal
+hlsl
+HMENU
+hmod
+hmodule
+hmon
+HMONITOR
+Horiz
+HORZ
+hostable
+hostlib
+Howett
+HPA
+HPAINTBUFFER
+HPCON
+hpj
+hpp
+HPR
+HPROPSHEETPAGE
+hread
+HREDRAW
+HREF
+hresult
+hrgn
+HRSRC
+hscroll
+hsl
+hstr
+hstring
+hsv
+HTCAPTION
+HTCLIENT
+htm
+HTMAXBUTTON
+HTMINBUTTON
+html
+HTMLTo
+HTTOP
+http
+hu
+hungapp
+HVP
+hwheel
+hwnd
+HWNDPARENT
+Hx
+hxx
+hypen
+hyperlink
+IAccessibility
+IAction
+IApi
+IApplication
+IBase
+icacls
```

Comment:
I had the idea that you could ignore anything in quotes. That way people could explicitly talk about something that they know is weird or typo'd or in a path like `icacls` and it could ignore the quoted as if it's a newspaper citation with [sic].

Reply:
The number of misspelled paths I've run across and the number of misspelled items in messages / quoted text...

Having the splitter look for `foo [sic]` and replace that w/ ` ` - would be doable.
I'd probably add a flag to support this behavior.

I think I'd be more inclined to offer:
```whitelist.txt
icacls 3 # ok in src/host/ft_integrity/IntegrityTest.cpp
```
Assuming I then went ahead and had the system keep track of the number of times it found a thing (it doesn't currently, but I've always wanted to do this, because it helps to know "this is an outlier"). And thus, if someone else added a fourth `icacls`, it'd complain (no number would mean unlimited).

Reply:
Hm... that's interesting too. I didn't think of leaving things a typo quota. Maybe in a future revision.